string robot_namespace
AABB bbox
RobotState state